hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/prefetch/TestS3ACachingBlockManager.java

```diff
@@ -125,13 +130,16 @@ public void testArgChecks() throws Exception {
   private static final class BlockManagerForTesting
       extends S3ACachingBlockManager {
 
+    private static final Configuration CONF = new Configuration();
```

Comment:
use prepareTestConfiguration() to isolate the temp dir. this always sets BUFFER_DIR so no need for the checks on L142